kapitan/inputs/jinja2_filters.py

```diff
@@ -47,6 +51,19 @@ def load_jinja2_filters(env):
     env.filters['shuffle'] = randomize_list
 
 
+def load_jinja2_filters_from_file(env, filter_paths):
+    for path in filter_paths:
+        try:
+            custom_filter_spec = util.spec_from_file_location(path.split('/')[-1].split('.')[0], path)
+            custom_filter_module = util.module_from_spec(custom_filter_spec)
+            custom_filter_spec.loader.exec_module(custom_filter_module)
+            for f in dir(custom_filter_module):
+                if isinstance(getattr(custom_filter_module, f), types.FunctionType):
```

Comment:
Move `getattr(custom_filter_module, f)` to a separate line to improve readability.